internal/world/world.go

```diff
@@ -83,7 +86,15 @@ func (w *World) Funcs() template.FuncMap {
 	funcs["vault"] = func(path, field string) (string, error) {
 		return w.Vault().Secret(path, field)
 	}
-
+	funcs["azure"] = func(path string) (string, error) {
+		return w.Azure().Secret(path)
+	}
```

Comment:
Is that short cut even needed? `{{ .Azure.Secret "..." }}` is much more readable IMO.

Reply:
Not sure i get it, you want the function to be called Azure.Secret? I wanted to keep in line with how we call vault now ("vault"). Ofc i am fine with chaning it

Reply:
Ok i updated it